### PR TITLE
Remote posts grid pattern

### PR DESCRIPTION
**Pull request by MaggieCabrera**
#### Changes proposed in this Pull Request:

Screenshot 2022-02-25 at 10 10 57

#### Related issue(s):

https://github.com/Automattic/themes/issues/5504

### CODE REVIEW

File: remote/inc/patterns/posts-grid.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	'content'    => '<!-- wp:group {"align":"full","layout":{"inherit":true}} -->
+	<div class="wp-block-group alignfull"><!-- wp:query {"queryId":0,"query":{"perPage":6,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true,"taxQuery":null},"tagName":"main","displayLayout":{"type":"flex","columns":2},"align":"wide"} -->
+	<main class="wp-block-query alignwide"><!-- wp:post-template -->
+	<!-- wp:separator {"className":"is-style-wide"} -->
+	<hr class="wp-block-separator is-style-wide"/>
+	<!-- /wp:separator -->
+	
+	<!-- wp:post-featured-image {"width":"100%","height":"220px","style":{"spacing":{"margin":{"top":"60px","bottom":"30px"}}}} /-->
+	
+	<!-- wp:post-date {"format":"M j","style":{"typography":{"textTransform":"uppercase"}},"fontSize":"small"} /-->
```
Comment (scruffian) [inline remote/inc/patterns/posts-grid.php]: I think post-date is always uppercase in this theme. Should we move this to theme.json?

File: remote/inc/patterns/posts-grid.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Posts grid.
+ *
+ * @package Remote
+ */
+
+return array(
+	'title'      => __( 'Posts grid', 'remote' ),
+	'categories' => array( 'pages' ),
```
Comment (scruffian) [inline remote/inc/patterns/posts-grid.php]: Should this be in Query?

File: remote/inc/patterns/posts-grid.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	'blockTypes' => array( 'core/query' ),
+	'content'    => '<!-- wp:group {"align":"full","layout":{"inherit":true}} -->
+	<div class="wp-block-group alignfull"><!-- wp:query {"queryId":0,"query":{"perPage":6,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true,"taxQuery":null},"tagName":"main","displayLayout":{"type":"flex","columns":2},"align":"wide"} -->
+	<main class="wp-block-query alignwide"><!-- wp:post-template -->
+	<!-- wp:separator {"className":"is-style-wide"} -->
+	<hr class="wp-block-separator is-style-wide"/>
+	<!-- /wp:separator -->
+	
+	<!-- wp:post-featured-image {"width":"100%","height":"220px","style":{"spacing":{"margin":{"top":"60px","bottom":"30px"}}}} /-->
```
Comment (scruffian) [inline remote/inc/patterns/posts-grid.php]: Should we make this a link?

**Comment by scruffian** [comment]
Is there anything we can do when not all posts have featured images?
Screenshot 2022-02-25 at 10 23 43

Also I think the post date could be a bit nearer the image...

**Comment by MaggieCabrera** [comment]
> Is there anything we can do when not all posts have featured images? 

I moved the margin from the image to a spacer and now at least the spacing is consistent:

Screenshot 2022-02-25 at 11 56 11



> 
> Also I think the post date could be a bit nearer the image...

I think that distance is correct, it needed to be more separated from the post title instead:
Screenshot 2022-02-25 at 11 53 04

**Review by scruffian — APPROVED** [review]
I added a commit. If you're good with it then I think we can merge.

**Comment by MaggieCabrera** [comment]
> I added a commit. If you're good with it then I think we can merge.

Sure thing!

**Pull request merged**